### PR TITLE
Add changes to use OpenSearch (instead of Elasticsearch) with security enabled

### PR DESCRIPTION
**Pull request by ramesh-maddegoda**
## 🗒️ Summary
Add changes to use OpenSearch (instead of Elasticsearch) with security enabled

## ♻️ Related Issues
Refer to task NASA-PDS/pds-registry-app#250

### CODE REVIEW

File: docker/.env [hunk 1]
```diff
@@ -49,7 +50,7 @@ TEST_DATA_URL=https://pds-gamma.jpl.nasa.gov/data/pds4/test-data/registry/urn-na
 HARVEST_DATA_DIR=./test-data/registry-harvest-data
 
 # Elasticsearch URL (the host name is the Elasticsearch service name specified in the docker compose)
-ES_URL=http://elasticsearch:9200
+ES_URL=https://elasticsearch:9200
```
Comment (nutjob4life) [inline docker/.env:53]: Why `https`?

Isn't this communication strictly internal to the host (within Docker)? This extra SSL would just increase CPU costs 🤔 

Is this a multi-host app? I thought it was all launched from a single `docker-compose.yaml`. I don't get it 🤷‍♀️

Comment (ramesh-maddegoda) [reply docker/.env:53]: @nutjob4life, I agree with you about the increased CPU costs.

The OpenSearch by default operates with security enabled. It is possible to disable security and run OpenSearch with http.  However, the purpose of this change was to make sure all our components can communicate with a secured OpenSearch (with https and username/password).

I would like to get opinions of @tloubrieu-jpl and @jordanpadams also on the point made by @nutjob4life .

Comment (jimmie) [reply docker/.env:53]: Testing https is necessary since the SAs will insist that the AWS OpenSearch service employ it.

Comment (nutjob4life) [reply docker/.env:53]: Oh so we will be using AWS-managed OpenSearch eventually … and this demonstrates we can use `https` to communicate with it.

Thanks for the explanation.

**Review by nutjob4life — APPROVED** [review]
Approved

**Review by tloubrieu-jpl — APPROVED** [review]
That wokrs with the batch loader so I will merge but I had an issue with the service loader, that might be on my side, see https://github.com/NASA-PDS/registry-harvest-service/pull/19#pullrequestreview-920707582

**Pull request merged**